templates/taxi/driver_list.html

```diff
@@ -1,14 +1,18 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-    <h1>
-      Driver List
-      <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
-        Create
-      </a>
-    </h1>
-
-    {% if driver_list %}
+  <h1>
+    Driver List
+    <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+      Create
+    </a>
+  </h1>
+  <form method="GET" action="" class="form-inline">
+    {{ search_form|crispy }}
+  <input class="btn btn-secondary" type="submit" value="🔍">
```

Comment:
From a usability perspective, it's better to have a label for each form input for accessibility reasons. Screen readers use these labels to provide context to the user. It's a good practice to always include a <label> for form controls.